api/v1/tile_set.config.js

```diff
@@ -62,6 +62,16 @@ module.exports = {
 		},
 		additionalProperties: false
 	},
+	sortingOptions: {
+		desc: {
+			column: 'created_on',
+			direction: 'desc'
+		},
+		asc: {
+			column: 'created_on',
+			direction: 'asc'
+		}
+	},
 	searchLimit: 50,
```

Comment:
yes, it's used as max `LIMIT` for `SELECT` query with search term. If a limit sent as request param is greater than `searchLimit ` the `searchLimit ` will be used